contracts/libraries/CoreLib.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.18;
 import "./SSVStorage.sol";
 
 library CoreLib {
-    event ModuleUpgraded(SSVModules moduleId, address moduleAddress);
+    event ModuleUpgraded(SSVModules indexed moduleId, address moduleAddress);
```

Comment:
This wont make any effect on the devs right?

Reply:
No, they can use it as usual. We can now filter by `moduleId` to search more easily when a module was upgraded. No need to add `msg.sender` as its implicit in the transaction.